backend/src/db/migrations/20240626111536_integration-auth-aws-assume-role.ts

```diff
@@ -0,0 +1,35 @@
+import { Knex } from "knex";
+
+import { TableName } from "../schemas";
+
+export async function up(knex: Knex): Promise<void> {
+  const hasAwsAssumeRoleCipherText = await knex.schema.hasColumn(
+    TableName.IntegrationAuth,
+    "awsAssumeIamRoleArnCipherText"
+  );
+  const hasAwsAssumeRoleIV = await knex.schema.hasColumn(TableName.IntegrationAuth, "awsAssumeIamRoleArnIV");
+  const hasAwsAssumeRoleTag = await knex.schema.hasColumn(TableName.IntegrationAuth, "awsAssumeIamRoleArnTag");
+  if (await knex.schema.hasTable(TableName.IntegrationAuth)) {
+    await knex.schema.alterTable(TableName.IntegrationAuth, (t) => {
+      if (!hasAwsAssumeRoleCipherText) t.text("awsAssumeIamRoleArnCipherText");
+      if (!hasAwsAssumeRoleIV) t.text("awsAssumeIamRoleArnIV");
+      if (!hasAwsAssumeRoleTag) t.text("awsAssumeIamRoleArnTag");
+    });
+  }
```

Comment:
what do you think about adding a metadata/methods JSON field in the IntegrationAuth table instead so that we could add integration auth-method specific details there? 

It might make sense to prepare for the future by making a generic field that can be reused by other auth methods for other integrations.

Reply:
There is already something called metadata. But technically this is not a metadata rather actual data. The whole integration table design needs a revamped and has quite some flaws including issues like using it in API etc. 

Something for the future i guess

Reply:
got this, yea that makes sense!